app/components/map/comparison/SideBySide.tsx

```diff
@@ -18,6 +18,8 @@ export default function SideBySide({
 }: SideBySideProps) {
   const map = useMap();
 
+  useEffect(() => {}, [leftDate, rightDate]);
```

Comment:
Is this necessary? You have `leftDate` and `rightDate` as dependencies in the `useEffect` below.

Reply:
removed